### ISSUE TITLE
D-pad doesn't work if using a joypad

### ISSUE THREAD

**Issue by sergiobenrocha2** [libretro/mame#43]
D-pad doesn't work if using a joypad, analog l-stick works as d-pad instead (quick menu --> controls --> User 1 Device Type = RetroPad).

Keyboard works fine

**Comment by r-type** [comment]
Ok i will look at when i can , i think i know why

**Comment by NelsonRosenberg** [comment]
Any news on this? Looks like it is still happpening

**Comment by AaronBPaden** [comment]
Have you tried hitting tab and remapping controls within mame?

**Comment by NelsonRosenberg** [comment]
I tried remapping inside the core with tab and even there the D-pad does not work. The other buttons all do, I can fire or jump, but not move. It works completely fine in non-mame cores and menu. Tried downloading standalone MAME and creating a default.cfg there with the mapping and pasting that cfg inside retroarch saves folder, overriding the one there, but again, it did not work. It looks almost like the core is not receiving any info about the d-pad from retroarch and it is not a configuration problem that we can tweak.
